app/views/hiring_staff/vacancies/_show.html.haml

```diff
@@ -0,0 +1,124 @@
+- content_for :page_title_prefix do
+  #{@vacancy.job_title} - #{@vacancy.school_name}
+
+.vacancy.grid-row
+  - if @vacancy.expired?
+    .column-full
+      .banner-warning
+        = t('jobs.expired')
+
+  .column-full
+    .share-url.display-none
+      %h2.heading-small.mt0 Share this job
+      #{link_to t('jobs.view_public_link'), @vacancy.share_url, id: 'share-url', 'aria-label': t('jobs.view_public_link')}, or #{link_to t('jobs.copy_public_url'), '#', class: 'copy-to-clipboard', 'data-clipboard-text': @vacancy.share_url, 'aria-label': t('jobs.copy_public_url')}
+    .share-url-no-js
+      %h2.heading-small.mt0 Share this job
+      #{link_to "#{t('jobs.view_public_link')} (#{@vacancy.share_url})", job_path(@vacancy), id: 'share-url'}
+
+  .column-two-thirds
+    %h1.heading-large
+      = @vacancy.job_title
+      %span.heading-secondary
+        = @vacancy.location
+
+    %h2.heading-medium= t('jobs.description')
+    %p= @vacancy.job_description
+
+    - if @vacancy.education?
+      %h2.heading-medium= t('jobs.education')
+      %p= @vacancy.education
+
+    - if @vacancy.qualifications?
+      %h2.heading-medium= t('jobs.qualifications')
+      %p= @vacancy.qualifications
+
+    - if @vacancy.experience?
+      %h2.heading-medium= t('jobs.experience')
+      %p= @vacancy.experience
+
+    - if @vacancy.benefits?
+      %h2.heading-medium= t('jobs.benefits')
+      %p= @vacancy.benefits
+
+    %hr
+    %h2.heading-medium
+      = t('schools.about')
+      = @vacancy.school.name
+
+    %p= @vacancy.school.description
+
+    %dl.school--details
+      %dt= t('schools.address')
+      %dd= @vacancy.school.full_address
+
+      - if @vacancy.school.phase.present?
+        %dt= t('schools.phase')
+        %dd= @vacancy.school.phase&.titleize
+
+      - if @vacancy.school.school_type.present?
+        %dt= t('schools.type')
+        %dd= @vacancy.school.school_type.label
+
+      - if @vacancy.school.url.present?
+        %dt= t('schools.website')
+        %dd
+          = link_to("#{@vacancy.school.name} website (opens in a new window)", @vacancy.school.url, class: 'wordwrap', target: '_blank')
```

Comment:
Can't we move all this to an `aria-label` instead of having it on the page visible to all users? It seems quite long winded.

Reply:
DAC advise semantic text for URLs, which I agree is a good idea for all users.